index.js

```diff
@@ -41,7 +49,17 @@ function getAllMovieTitles() {}
  *  getHighestMetascore(movies);
  *  //> 96
  */
-function getHighestMetascore() {}
+function getHighestMetascore(movies) {
+  let highMeta = 0;
```

Comment:
Maybe `highestMetascore` here. `highMeta` sounds sorta like the new Facebook parent company subsidiary 😆